QModManager/Utility/Logger.cs

```diff
@@ -22,7 +22,7 @@ internal enum Level
         /// <value>
         ///   <c>true</c> if debug logs are enabled; otherwise, <c>false</c>.
         /// </value>
-        private static bool EnableDebugLogging { get; } = File.Exists(Path.Combine(Environment.CurrentDirectory, "QModDebug.txt"));
+        internal static bool EnableDebugLogging { get; } = File.Exists(Path.Combine(Environment.CurrentDirectory, "QModDebug.txt"));
```

Comment:
Will this still need to be `internal` after https://github.com/SubnauticaModding/QModManager/pull/125 is merged?

Reply:
No, this will be completely removed.